docs/docs/native-docker.md

```diff
@@ -36,20 +36,19 @@ DCOS users do not need to follow the steps below.
     $ echo 'docker,mesos' > /etc/mesos-slave/containerizers
     ```
 
-2. Increase the executor timeout to account for the potential delay in pulling a docker image to the slave.
-
+2. Increase the executor timeout to account for the potential delay pulling a docker image to the agent node.
 
     ```bash
-    $ echo '5mins' > /etc/mesos-slave/executor_registration_timeout
+    $ echo '10mins' > /etc/mesos-slave/executor_registration_timeout
     ```
 
-3. Restart `mesos-slave` process to load the new configuration
+3. Restart the `mesos-slave` process to load the new configuration.
```

Comment:
Understand but in [Mesos docs](http://mesos.apache.org/documentation/latest/agent-recovery/) this process is referred as `mesos-agent`.